docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -92,6 +92,9 @@ journey('Journey name', ({ page, browser, context, params }) => {
                 For example, if you want to use a different homepage depending on the `env`
                 (`localhost` for `dev` and a URL for `prod`). See <<synthetics-params-secrets>>
                 for more information.
+`request`::     A request object that can be used to make API requests independently of the browser
```

Comment:
We want to mention how this is different from the other request objects that is exposed by Playwright. 

```
1. request
2. context.request
3. page.request
```

We should refer to this docs - https://playwright.dev/docs/api/class-apirequestcontext

Reply:
++, I think a link out is plenty, as long as it's clear which part of the docs applies.